modules/nw-kuryr-cleanup.adoc

```diff
@@ -0,0 +1,280 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.adoc
+
+:_content-type: PROCEDURE
+[id="nw-kuryr-cleanup_{context}"]
+= Cleaning up resources after migration
+
+After migration from the Kuryr network plugin to the OVN-Kubernetes network plugin completes, clean up the resources that Kuryr created previously. 
+
+.Prerequisites
+
+* You installed the {product-title} CLI, `oc`.
+* You installed a Python interpreter.
+* You installed the `openstacksdk` Python package.
+* You installed the `openstack` CLI.
+* You have access to the underlying {rh-openstack} cloud.
+* You can access the cluster as a user with the `cluster-admin` role.
+
+[NOTE]
+====
+At the time of writing when OCP16 is used, it includes ancient version of
+`openstacksdk` package, that does not support tags for Octavia objects. It is
+recommended to use either workstation with modern operating system with newer
+version of `openstacksdk` package or `virtualenv` and install appropriate
+version from link:https://pypi.org[pypi].
+====
+
+.Setup virtualenv
+
+If you decide to use virtualenv, here is a quick guidance how to proceed.
+At the time of writing, minimal versions of the following OpenStack components
+has been used to fulfill cleaning procedure:
+
+- openstacksdk 0.54.0
+- python-openstackclient 5.5.0
+- python-octaviaclient 2.3.0
+
+Newer versions should also work, although there is a tiny chance for
+incompatibilities. If you have issues, either downgrade to version mentioned
+above or consult individual component documentation and/or changelog for
+details.
+
+To set up cleanup environment, create virtualenv, update pip and install needed
+components:
+
+[source,terminal]
+----
+$ python3 -m venv /tmp/venv
+$ source /tmp/venv/bin/activate
+(venv) $ pip install pip --upgrade
+(venv) $ pip install openstacksdk==0.54.0 python-openstackclient==5.5.0 python-octaviaclient==2.3.0
+----
+
+[NOTE]
+====
+In the command prompt there will be visible indicator, that virtualenv is in
+use - here it will be additional `(venv)` string, as in the example above venv
+is a name of the directory and the virtual env at the same time.
+====
+
+Stay in the created environment during following procedure to be sure the
```

Comment:
@maxwelldb everything below this point (based on this message) should start with:  `(venv) $`